pkg/client/v1/model/NsALUA.go

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 Hewlett Packard Enterprise Development LP
+ */
```

Comment:
The year is off and it doesn't follow the typical comment pattern for copyright

Reply:
Will fix it . thanks